input/parameters/lithium-ion/example_function.py

```diff
@@ -0,0 +1,11 @@
+#
+# Example function, for testing
+#
+from __future__ import absolute_import, division
+from __future__ import print_function, unicode_literals
+
+import numpy as np
+
+
+def example_function(x):
+    return x * np.exp(x - 1)
```

Comment:
I guess we don't need this file any more?

Reply:
Good point!